resources/metadata/abapEnvironmentRunATCCheck.yaml

```diff
@@ -0,0 +1,101 @@
+metadata:
+  name: abapEnvironmentRunATCCheck
+  description: Runs an ATC Check
+  longDescription: |
+    Run ATC Check
+spec:
+  inputs:
+    secrets:
+      - name: credentialsId
+        description: Jenkins credentials ID containing user and password to authenticate to the Cloud Foundry API
+        type: jenkins
+    params:
+      - name: atcConfig
```

Comment:
Is this a path to a file?

Reply:
Path to a YAML configuration file. I further clarified the description.